cms/models.py

```diff
@@ -27,6 +28,35 @@
 from main.views import _serialize_js_settings
 
 
+class HomePage(Page):
```

Comment:
I think better to structure the `HomePage` such as to capable for using some common properties from `ProductPage` instead of replicating again.
Better to define a parent for this here. 
```
Class CommonProperties(object):       
              .... 
```